.github/workflows/main.yaml

```diff
@@ -3,7 +3,7 @@ name: NNF Test
 on: [push]
 
 jobs:
-  test:
+  int-test:
```

Comment:
If we are not yet ready to satisfy this job that runs on the self-hosted environment, then maybe it's time to disable it until we are ready.

Reply:
I almost did as part of this PR. I agree. I'll do that in another PR.

Reply:
Never mind. I made it part of this one. I split the jobs into separate workflows and then we can turn off int-test manually using the web UI.